src/main/java/dk/bec/polonez/reservationsystem/dto/userDto/ProfileResponse.java

```diff
@@ -0,0 +1,25 @@
+package dk.bec.polonez.reservationsystem.dto.userDto;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class ProfileResponse {
+
+    private Long id;
+
+    private String username;
+
+    private String name;
+
+    private String email;
+
+    private String roleName;
```

Comment:
**advice** What about using ```enum``` for specifying a finite list of allowed role names?

Reply:
currently, our solution defines roles in database, so my team would have to rearrange the code to merge it with the enum. As we don't have much time, it can be an idea with lower priority.